frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -32,15 +41,32 @@ const dashReducer = (state: DashState, action: DashAction): DashState => {
   }
 };
 
+const timelineReducer = (state: TimelineState, action: TimelineAction): TimelineState => {
+  switch (action.type) {
+    case "UPDATE": {
```

Comment:
this is probably minor and can be updated later but do we need more granular updates (e.g. updating based on time segment versus filtering by services or event type?